### PR TITLE
Update baselines for recent changes in F3D

### PR DESCRIPTION
**Pull request by mwestphal**

### CODE REVIEW

File: testing/baselines/TestOCCT.png
Comment (michaelmigliore) [inline testing/baselines/TestOCCT.png:1]: What's wrong with the colors?

Comment (Meakk) [inline testing/baselines/TestOCCT.png:1]: what's wrong with the colors?

Comment (mwestphal) [reply testing/baselines/TestOCCT.png:1]: macOS coloring is wrong with OCCT data, no idea why. It was already the case before, it was just not as visible.

Comment (mwestphal) [reply testing/baselines/TestOCCT.png:1]: ha wait, it should not be colored at all, need to check again

Comment (mwestphal) [reply testing/baselines/TestOCCT.png:1]: fixed

File: testing/CMakeLists.txt [hunk 1]
```diff
@@ -87,7 +87,7 @@ if (draco_enabled)
 endif ()
 
 if (occt_enabled)
-  f3d_add_test(TestOCCT f3d.stp --cells=0) # A simple OCCT test that work whether coloring is turned on or not
+  f3d_add_test(TestOCCT f3d.stp --scalars=f3d_reserved) # A simple OCCT test that work whether coloring is turned on or not
```
Comment (Meakk) [inline testing/CMakeLists.txt:90]: isn't `--scalars` enough?

Comment (mwestphal) [reply testing/CMakeLists.txt:90]: --scalars turns it on, no way to turn it off, see https://github.com/f3d-app/f3d/issues/837

**Review by Meakk — APPROVED** [review]

**Pull request merged**